pkgs/development/ocaml-modules/csexp/default.nix

```diff
@@ -1,24 +1,30 @@
-{ lib, fetchurl, buildDunePackage, result }:
+{ lib, fetchurl, buildDunePackage, liquidsoap }:
 
 buildDunePackage rec {
   pname = "csexp";
-  version = "1.5.1";
+  version = "1.5.2";
 
   src = fetchurl {
     url = "https://github.com/ocaml-dune/csexp/releases/download/${version}/csexp-${version}.tbz";
-    sha256 = "sha256-1gXkBl+pCliABEDvLzOi2TE5i/LCIGGorLffhFwKrAI=";
+    hash = "sha256-GhTdBLtDeaQZkCSFUGKMd5E6nAfzw1wTcLaWDml3h/8=";
   };
 
-  minimumOCamlVersion = "4.03";
-  useDune2 = true;
+  postPatch = ''
+    substituteInPlace dune-project --replace "dune 3.4" "dune 2.0"
```

Comment:
I think we should just hold off on merging this until we can move it to dune 3 instead. Unless @vbgl is of another opinion.